DPG/Tasks/AOTTrack/V0Cascades/perfK0sResolution.cxx

```diff
@@ -127,6 +151,39 @@ struct perfK0sResolution {
     if (enableTPCPlot) {
       rK0sDauResolution.add("h3_tpc_vs_pid_hypothesis", "h3_tpc_vs_pid_hypothesis", {HistType::kTH3F, {{200, -10.f, 10.f, "#it{p}/Z (GeV/#it{c})"}, {1000, 0, 1000.f, "dE/dx (a.u.)"}, {10, -0.5, 9.5f, "PID hypothesis"}}});
     }
+
+    ccdb->setURL(ccdburl);
+    ccdb->setCaching(true);
+    ccdb->setLocalObjectValidityChecking();
+    ccdb->setFatalWhenNull(false);
+
```

Comment:
if you use it only for the track tuner then you can put it in the scope of `if (useTrackTuner)`